app/src/main/java/org/sopt/official/feature/auth/feature/certificatemember/CertificateMemberScreen.kt

```diff
@@ -0,0 +1,231 @@
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package org.sopt.official.feature.auth.feature.certificatemember
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.border
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.SnackbarHost
+import androidx.compose.material3.SnackbarHostState
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.rememberCoroutineScope
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import kotlinx.coroutines.launch
+import org.sopt.official.R.drawable.ic_auth_certification_error
+import org.sopt.official.R.drawable.ic_auth_memeber_error
+import org.sopt.official.R.drawable.ic_auth_process
+import org.sopt.official.designsystem.Blue500
+import org.sopt.official.designsystem.BlueAlpha100
+import org.sopt.official.designsystem.Gray10
+import org.sopt.official.designsystem.Gray100
+import org.sopt.official.designsystem.Gray60
+import org.sopt.official.designsystem.Gray80
+import org.sopt.official.designsystem.Gray950
+import org.sopt.official.designsystem.Red100
+import org.sopt.official.designsystem.SoptTheme
+import org.sopt.official.designsystem.White
+import org.sopt.official.feature.auth.component.AuthButton
+import org.sopt.official.feature.auth.component.AuthTextField
+import org.sopt.official.feature.auth.component.AuthTextWithArrow
+import org.sopt.official.feature.auth.component.CertificationSnackBar
+import org.sopt.official.feature.auth.component.PhoneCertification
+
+@Composable
+fun CertificateMemberScreen() {
+    val snackBarHostState = remember { SnackbarHostState() }
+    val coroutineScope = rememberCoroutineScope()
+    val onShowSnackBar: () -> Unit = {
+        coroutineScope.launch {
+            snackBarHostState.showSnackbar("인증번호가 전송되었어요.")
+        }
+    }
```

Comment:
```suggestion
    val onShowSnackBar: () -> Unit = remember { {
        coroutineScope.launch {
            snackBarHostState.showSnackbar("인증번호가 전송되었어요.")
        }
    } }
```

(혹시 모르는데) 이 함수가 리컴포지션될때마다 매번 리컴포지션 범위에 포함되어서 다시 함수객체가 만들어질 수 있으니 remember로 감싸주면 좋을 것 같아요